wit/deps.toml

```diff
@@ -1 +1 @@
-io = "https://github.com/wasmCloud/wasi-io/archive/main.tar.gz" # TODO: Switch to upstream once compatible with Wasmtime 12
+io = "https://github.com/WebAssembly/wasi-io/archive/main.tar.gz"
```

Comment:
This is worth celebrating! 🎉

Reply:
Archiving `wasmCloud/wasi-io` might be a good way to start!

Reply:
@connorsmith256 may be able to do that, but I do not have the necessary permissions for that

Reply:
Done! Archived 🧹